doc/mapbender_terminal_commands.md

```diff
@@ -0,0 +1,315 @@
+
+> ComposerBootstrap::prepareBinaries
+> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
+```
+
+#### Update individual or specific packages
+
+To limit the upgrade process to a few or specific packages, you can use the packages that you want to update, specify as such:
+ 
+```bash
+php composer.phar update mapbender/mapbender mapbender/data-source
+```
+
+* example:
+
+```text
+/application$: php composer.phar update mapbender/mapbender mapbender/data-source
+Loading composer repositories with package information                                                                                         Updating dependencies (including require-dev)         Package operations: 0 installs, 1 update, 0 removals
+  - Updating mapbender/mapbender dev-release/3.0.6 (9fde835 => ee7a6bc):  Checking out ee7a6bc1fc
+Package components/datatables is abandoned, you should avoid using it. Use bower-asset/datatables instead.
+Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
+Writing lock file
+Generating optimized autoload files
+> ComponentInstaller\Installer::postAutoloadDump
+Compiling component files
+> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
+> ComposerBootstrap::installAssets
+
+ Trying to install assets as relative symbolic links.
+
+ --- ----------------------------------- ------------------ 
+      Bundle                              Method / Error    
+ --- ----------------------------------- ------------------ 
+  ✔   FrameworkBundle                     relative symlink  
+  ✔   FOSJsRoutingBundle                  relative symlink  
+  ✔   FOMCoreBundle                       relative symlink  
+  ✔   FOMManagerBundle                    relative symlink  
+  ✔   FOMUserBundle                       relative symlink  
+  ✔   MapbenderCoreBundle                 relative symlink  
+  ✔   MapbenderWmcBundle                  relative symlink  
+  ✔   MapbenderWmsBundle                  relative symlink  
+  ✔   MapbenderManagerBundle              relative symlink  
+  ✔   MapbenderPrintBundle                relative symlink  
+  ✔   MapbenderMobileBundle               relative symlink  
+  ✔   MapbenderDigitizerBundle            relative symlink  
+  ✔   MapbenderRoutingBundle              relative symlink  
+  ✔   MapbenderCoordinatesUtilityBundle   relative symlink  
+  ✔   SensioDistributionBundle            relative symlink  
+ --- ----------------------------------- ------------------ 
+
+ [OK] All assets were successfully installed.                                                                           
+
+> ComposerBootstrap::prepareBinaries
+> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
+
+```
+
+#### Update without dependencies
+
+If only the dev-require packages are to be updated, then you can optionally specify the parameter `--no-dev`. It disables the installation of require-dev packages.
```

Comment:
Doesn't --no-dev skip packages listed under 'require-dev'? If so, it should be "If only the non-dev/production dependencies are to be updated..."

Reply:
Thanks. Right.